php/php.editor/test/unit/data/testfiles/actions/testGH5578/01/testGH5578_01.php.fixUses

```diff
@@ -0,0 +1,39 @@
+<?php 
+
+use NS1\B;
+declare (ticks=1) {
+    
+    
+}
```

Comment:
We should add use statements below the declare statement if possible.
see: https://www.php-fig.org/psr/psr-12/#3-declare-statements-namespace-and-import-statements

Please add a test case that has also `namespace` (`declare` and `namespace`).

Reply:
Updated logic due to it, added new tests, also added a check for USE placing after phpDoc if there is no namespace in file (if namespace and phpDoc are there it will be placed after namespace).